src/amo/pages/Guides/index.js

```diff
@@ -35,6 +36,7 @@ type InternalProps = {|
   addons: {
     [guid: string]: AddonType | null,
   },
+  clientApp: ?string,
```

Comment:
This prop should always be defined, no?

Reply:
This was the only way to make flow happy if I recall...so hopefully this is ok..?